### PR TITLE
(DOCSP-3589) Document custom collation in Create Database, Collection…

### PR DESCRIPTION
**Pull request by rob-mongodb**
…, and Index modals.

Staged:

- https://docs-mongodbcom-staging.corp.mongodb.com/compass/robertjustice/DOCSP-3588/indexes.html#create-an-index

- https://docs-mongodbcom-staging.corp.mongodb.com/compass/robertjustice/DOCSP-3588/collections.html#create-a-collection

- https://docs-mongodbcom-staging.corp.mongodb.com/compass/robertjustice/DOCSP-3588/databases.html#create-a-database

### CODE REVIEW

**Comment by rob-mongodb** [comment]
This is for DOCSP-3589, not DOCSP-3588, which is a related ticket.

File: source/includes/steps-create-collection.yaml [hunk 1]
```diff
@@ -6,14 +6,18 @@ content: |
...
 ref: enter-collection-name
 content: |
    In the dialog, enter the name of the collection to create.
 
-   If you want to create a capped collection, select the
-   :guilabel:`Capped Collection` checkbox and enter the maximum bytes.
+   If you want to create a :manual:`capped collection </core/capped-collections>`,
+   select the :guilabel:`Capped Collection` checkbox and enter the maximum bytes.
+
+   If you want to use `custom collation <https://docs.mongodb.com/manual/indexes/index.html#indexes-and-collation>`_,
```
[the remark this answers is not in the document]
Comment (rob-mongodb) [reply source/includes/steps-create-collection.yaml]: Substituted your link.  Would rather not duplicate the information from the new link target.  Fixed.

File: source/includes/steps-create-collection.yaml [hunk 2]
```diff
@@ -6,14 +6,18 @@ content: |
...
+   select the :guilabel:`Use Custom Collation` checkbox and select the
+   desired collations.
```
[the remark this answers is not in the document]
Comment (rob-mongodb) [reply source/includes/steps-create-collection.yaml]: Good suggestion.  Fixed.

File: source/includes/steps-create-database.yaml [hunk 3]
```diff
@@ -7,16 +7,20 @@ content: |
...
    In the dialog, enter the name of the database to create and its
    first collection. Both the database name and the collection name are
    required.
 
-   If you want to create a capped collection, select the
-   :guilabel:`Capped Collection` checkbox and enter the maximum bytes.
+   If you want to create a :manual:`capped collection </core/capped-collections>`,
+   select the :guilabel:`Capped Collection` checkbox and enter the maximum bytes.
+
+   If you want to use `custom collation <https://docs.mongodb.com/manual/indexes/index.html#indexes-and-collation>`_,
```
[the remark this answers is not in the document]
Comment (rob-mongodb) [reply source/includes/steps-create-database.yaml]: Fixed.

File: source/includes/steps-create-database.yaml [hunk 4]
```diff
@@ -7,16 +7,20 @@ content: |
...
+   select the :guilabel:`Use Custom Collation` checkbox and select the
+   desired collations.
```
[the remark this answers is not in the document]
Comment (rob-mongodb) [reply source/includes/steps-create-database.yaml]: Fixed.

File: source/includes/steps-create-index.yaml [hunk 5]
```diff
@@ -20,10 +20,22 @@ content: |
...
    To index additional fields, click :guilabel:`Add Another Field`.
 ---
-title: Specify the index options, if any.
+title: Optional. Specify the index options, if any.
 level: 4
 ref: index-options
-content: ""
+content: |
+
+   Compass supports the following index options:
```
[the remark this answers is not in the document]
Comment (rob-mongodb) [reply source/includes/steps-create-index.yaml]: Fixed.

File: source/includes/steps-create-index.yaml [hunk 6]
```diff
@@ -20,10 +20,22 @@ content: |
...
+
+   * `Build the index in the background <https://docs.mongodb.com/master/core/index-creation/index.html#background-construction>`_
+
+   * :manual:`Create a unique index </core/index-unique>`
+
+   * :manual:`Create a TTL Index </core/index-ttl>`
+
+   * :manual:`Create a partial index </core/index-partial/>`
+
+   * `Use custom collation <https://docs.mongodb.com/manual/indexes/index.html#indexes-and-collation>`_
```
[the remark this answers is not in the document]
Comment (rob-mongodb) [reply source/includes/steps-create-index.yaml]: Fixed.

**Comment by rob-mongodb** [comment]
@jeff-allen-mongo , ready for another look.

File: source/includes/steps-create-index.yaml [hunk 7]
```diff
@@ -20,10 +20,24 @@ content: |
...
+
+     For information on the Custom Collation settings, see `Collation <https://docs.mongodb.com/manual/reference/collation/#collation-document>`_.
```
[the remark this answers is not in the document]
Comment (rob-mongodb) [reply source/includes/steps-create-index.yaml]: Good idea.  Fixed.

**Comment by alenakhineika** [comment]
LGTM

**Pull request merged**